### PR TITLE
feat: new dashboard widget's option selection

### LINKED ISSUE #947: Value based widget actions overlap with panel title
## Bug description

*Please describe.*  
Value type widget which are added to dashboards have 3 actions buttons which overlap with that panel title.
Screenshot 2022-04-03 at 20-11-27 Open source Observability platform SigNoz

## Expected behavior

It should not overlap, simplest solution would be to introduce three dot menu / kebab menu style.

## Version information 
* **Signoz version**: 0.7.4
* **Browser version**: Firefox 98.0.2 (64-bit)
* **Your OS and version**:macOS Monterey 12.3 
* **Your CPU Architecture**(ARM/Intel): ARM

### PR DESCRIPTION
**Pull request by pranshuchittora**
Closes #947 

https://user-images.githubusercontent.com/32242596/162756010-0466f4b0-4979-4b00-b503-e0a6d7c7ada8.mov

### CODE REVIEW

**Comment by palash-signoz** [comment]
@pranshuchittora can we use `type='danger'` for delete ?

**Comment by palash-signoz** [comment]
apart from that LGTM

**Review by palash-signoz — CHANGES_REQUESTED** [review]
@pranshuchittora can we use type='danger' for delete ?

**Comment by pranshuchittora** [comment]
> @pranshuchittora can we use type='danger' for delete ?

Screenshot 2022-04-15 at 12 04 52


@pranay01 can you pls confirm whether the delete should be of type danger or not. I agree with Palash but just confirming because this deviates from the provided design.

**Comment by pranay01** [comment]
@pranshuchittora yeah makes sense. lets use type='danger' for delete

File: frontend/src/container/GridGraphLayout/Graph/index.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
```
Comment (palash-signoz) [inline frontend/src/container/GridGraphLayout/Graph/index.tsx:1]: Can you remove this eslint disable rule and attach a keyboard listener

**Review by palash-signoz — CHANGES_REQUESTED** [review]

File: frontend/src/container/GridGraphLayout/Graph/index.tsx (hunk 1 above)
Comment (pranshuchittora) [reply frontend/src/container/GridGraphLayout/Graph/index.tsx]: Done

**Review by palash-signoz — APPROVED** [review]

**Pull request merged**